### PR TITLE
[IE][Myriad] LSTMCell: check number of paramters

### PR DESCRIPTION
**Pull request by elatkin**
Accompanies MDK #[14460](https://github.com/movidius/mdk/pull/14660)

### CODE REVIEW

File: inference-engine/src/vpu/graph_transformer/src/stages/rnn.cpp [hunk 1]
```diff
@@ -83,6 +83,18 @@ class LSTMCellStage final : public StageNode {
         bool useTempBuffer = (nCells > 1);
         IE_ASSERT((numTempBuffers() == 1 && useTempBuffer) || !useTempBuffer);
 
+        VPU_THROW_UNLESS(inputEdges().size() == 5,
+                         "input edges: {}", inputEdges().size());
```
Comment (gladilov-gleb) [inline inference-engine/src/vpu/graph_transformer/src/stages/rnn.cpp:87]: Please provide more descriptive message. From message "input edges: 3" the intention is unclear. Is it too big, too small or what?

Comment (elatkin) [reply inference-engine/src/vpu/graph_transformer/src/stages/rnn.cpp:87]: Obviously, it must equal to 5....

Or do you mean, that VPU_THROW_UNLESS would not print the assertion expression?

Comment (gladilov-gleb) [reply inference-engine/src/vpu/graph_transformer/src/stages/rnn.cpp:87]: I'm not sure about that, anyway it's better not to rely on it, In your PR a little bit lower you provided expected value in the message, though

Comment (elatkin) [reply inference-engine/src/vpu/graph_transformer/src/stages/rnn.cpp:87]: OK, now I see: VPU_THROW_UNLESS does not print the assert expression. 😞 

So I will provide more descriptive message here with my next commit.

Thanks for pointing to this!

**Review by gladilov-gleb — APPROVED** [review]

**Comment by gladilov-gleb** [comment]
@elatkin please fill the Milestone field

**Review by andrejsokolov — APPROVED** [review]

**Comment by elatkin** [comment]
@gladilov-gleb 
Could you please merge this pull-request?

Please note: I have added switching to new VPU firmware since you have approved this pull-request
Results of additional ICV Jenkins/CI testing are found via comments to accompanying MDK #[14660](https://github.com/movidius/mdk/pull/14660)

**Review by gladilov-gleb — APPROVED** [review]

**Pull request merged**